### DISCUSSION TITLE
Low amount DARs, wrong latent.vars

### DISCUSSION THREAD

**Discussion by JoreVW** [stuart-lab/signac#1557] [Q&A] [answered]
Dear Tim,

I have a question on finding DARs. I have multiome GEX + ATAC data, two samples that I merged and integrated following the vignettes and tips I found here at the repository. Recalled the peaks with MACS2 and also made a combined set of peak upon merging the samples. Just wanted to highlight everything was done according to the vignette and the tips I found here.

My data looks good, however when I look for DARs the number is very low, especially compared to DEGs. 

My data is from the heart, I subsetted the cardiomyocyte population as a new object, reclustered to find some differences in the cardiomyocyte population specifically. My two samples have either WT as identity or KO as identity.

As an exemple, if I want to check for the cardiomyocyte population in general the DEGs and DARs between KO and WT, I find +1000 DEGs, but only 500 DARs. When looking at the peaks, the DARs are called correctly, I can see an obvious difference in peak hight. However, I also see peaks that are obviously different in hight but not called as a DAR. So I have the feeling I'm using the wrong latent.vars?

Initially the code would be;

da_peaks2 <- FindMarkers(
  object = CM_subset,
  assay = "peaks",
  ident.1 = "ko",
  ident.2 = "wt",
  min.pct = 0.05,
  test.use = 'LR',
  logfc.threshold = 0,
  only.pos = F,
  latent.vars = 'nCount_peaks')

Even with a min.pct of 0.05 and a logic.threshold of 0, I only get 500 DARs for the complete population of cardiomyocytes. 
I really think I'm using a wrong setting, I'm being to strict somewhere because the few DARs I find, are still correct. 

I tried to replace "nCount_peaks" by "genotype", which basically represents the samples in my dataset (KO, WT) but than all my p values are 1.

I have seen people using "atac_peak_region_fragments" as input, but this is also just the fragment count per barcode right? so same as nCount_peaks.

Please let me know if you need some additional input (session info).
I have tried to look in this repository for a same problem, but could not really find it. My apologies if it did, can you please refer me then.

Thanks,

Jore

**Comment by JoreVW** [comment]
the difference with #1171 Is that my FC are high or low enough, (4, -4)

**Comment by timoast** [comment, marked as answer]
If there is a difference in total counts between the WT and KO conditions then that could explain why some changes are not statistically significant. If you want to find a more extensive list of peaks just based on fold change, you can run the `FoldChange()` function. Keep in mind that this does not run any statistical test. Alternatively, if your data is very sparse you could try setting `min.pct` lower.

**Reply by JoreVW** [reply]
There is indeed a difference in the total counts between the 2 condition. Thank you, this makes sense, I did not think of this.

**Reply by JoreVW** [reply]
@timoast I checked the sparsity of the data and it is not very sparse compared to what to expect from atac. To be sure I ran a lower min.pct (0.01) , and I only found a handful of DARs more compared to 0.05.

So the issue will indeed be the big difference in total counts. 

Do you think another built-in test of FindMakers would be better here? I have looked in the repositories and I understand the reason you (and others recommend LR). I'm checking with MAST now and contact them but maybe you have encountered a similar problem already? I have found one tool, PeakVI, that take it another way. I'll try to figure it out to implement on my dataset, but maybe you have encountered this issue in the past?

Thanks again for your quick respons on this matter, as always!

**Reply by timoast** [reply]
If the total counts correlates with the condition you're testing, then there is confounding which will be difficult to deal with statistically. You can ignore the differences in depth but then you won't be confident that the changes between conditions are due to KO and not technical factors (sequencing depth). Not sure what to suggest here other than add replicates or downsample depth to the same level across condition, or add reads to the condition with less depth
